index.html

```diff
@@ -29,44 +29,44 @@ <h6> Profile</h6></a>
     </nav>
     <main class="main-login main-home">
         <div class="offer-image">
-            <img src="/assets/offer-img-main-2.png" alt="">
+            <a href="./pages/product_list_page/products_list.html"><img src="/assets/offer-img-main-2.png" alt=""/></a>
         </div>
         <h5 class="brand-heading"> UNBELIEVABLE DEALS @ min.50% OFF</h5>
         <div class="sale-img-container">
-            <img src="/assets/sale-pic-1.png" alt="">
-            <img src="/assets/sale-pic-2.png" alt="">
-            <img src="/assets/sale-pic-3.png" alt="">
-            <img src="/assets/sale-pic-4.png" alt="">
-            <img src="/assets/sale-pic-5.png" alt="">
+            <a href="./pages/product_list_page/products_list.html"><img src="/assets/sale-pic-1.png" alt=""/></a>
+            <a href="./pages/product_list_page/products_list.html"><img src="/assets/sale-pic-2.png" alt=""/></a>
+            <a href="./pages/product_list_page/products_list.html"><img src="/assets/sale-pic-3.png" alt=""/></a>
+            <a href="./pages/product_list_page/products_list.html"><img src="/assets/sale-pic-4.png" alt=""/></a>
+            <a href="./pages/product_list_page/products_list.html"><img src="/assets/sale-pic-5.png" alt=""/></a>
         </div>
         <h5 class="brand-heading"> TOP-BRANDS</h5>
         <div class="top-brand-container">
-            <img src="/assets/sale-pic-6.png" alt="">
-            <img src="/assets/sale-pic-7.png" alt="">
-            <img src="/assets/sale-pic-8.png" alt="">
-            <img src="/assets/sale-pic-9.png" alt="">
-            <img src="/assets/sale-pic-10.png" alt="">
+            <a href="./pages/product_list_page/products_list.html"><img src="/assets/sale-pic-6.png" alt=""/></a>
```

Comment:
you can write some name alt attributes

Reply:
But they are images which have to change on sales everytime..hence i left it like that only....Thank you for your suggestion ❤️